src/util/XDRCereal.h

```diff
@@ -51,6 +51,27 @@ cereal_override(cereal::JSONOutputArchive& ar,
     xdr::archive(ar, tmp, field);
 }
 
+template <uint32_t N>
+void
+cereal_override(cereal::JSONOutputArchive& ar,
```

Comment:
I don't think this is necessary once https://github.com/xdrpp/xdrpp/pull/25 is merged.

Reply:
After several attempts, I think I'll give up fixing this in a more general way such as a new `save` function in XDRPP or `cereal_override` for `xvector`. While I definitely think that it's suboptimal that we need to implement something like this whenever we have an `xvector` containing elements with `cereal_override`, it seems that this is our second time, so I hope that this is not that bad. Let me know what you think!